lib/rdl/util.rb

```diff
@@ -1,3 +1,5 @@
+require 'stringio'
```

Comment:
@jeffrey-s-foster The goodcheck experiment was crashing without this.

Reply:
Why?

Reply:
Without it, it prints:

```
lib/rdl/util.rb:94:in `silent_warnings': uninitialized constant RDL::Util::StringIO (NameError)
```

The method in question:
```ruby
  def self.silent_warnings
    old_stderr = $stderr
    $stderr = StringIO.new
    yield
  ensure
    $stderr = old_stderr
  end
```